doc/contributing/reconizing-contributors.md

```diff
@@ -20,6 +20,24 @@ Some of the benefits we hope to achieve through these programs include:
   done by individuals or teams like those mentioned. This should
   encourage other people to get involved and make similar contributions
 
+## Sponsorship link on nodejs/node README.md
+
+Collaborators may add a single link beside their name on the nodejs/node
+README.md page named `Support me`. This link may be:
+
+1. a direct link to one of the sponsorship platforms listed below
+2. a page which has text explaining how to support the collaborator
+   including zero or more links to the sponsorship platforms listed below.
+   The page shoud not have any outgoing links other than to the
+   sponsorship platforms. We expect and trust collaborators to ensure that
+   the text is professional and repectful.
```

Comment:
```suggestion
2. a url which has text explaining how to support the collaborator
   including zero or more links to the sponsorship platforms listed below.
   We expect and trust collaborators to ensure that
   the text and all the outgoing links are professional and repectful.
```

In my personal case, I am more interesting in building a personal site that explains how to support my work in Open Source like (support.ulisesgascon.com or similar)  and I probably will promote my own books that are for sale, referral links to platforms such as Digital Ocean or contributions that I did in the past, etc... So, with the existing definition I am limited only to add links to sponsorship platforms but not to other outgoing links.

I believe that the links should be as professional and respectful as the text included.

Side Note: I am not yet using sponsorship platforms as I need to investigate how the taxes will work for it in my local context (Spain)

Reply:
I think the idea is to forbid to link back to your sponsors – which might be worth stating explicitly, as I think it can be easily overlooked.

Reply:
> I think the idea is to forbid to link back to your sponsors

@aduh95 I'm not sure I follow.

@UlisesGascon I do understand your concern. I was trying to address earlier concerns that were raised about pointing to sponsorship options that would reflect poorly on the project.  My suggestion for one page + defined outoing links was to address that but open to other suggestions as well.